### ISSUE TITLE
How to get QueryInst to train on an external dataset

### ISSUE THREAD

**Issue by fbragman** [hustvl/QueryInst#50]
Hi,

I am trying to train QueryInst with a Swin backbone on my own dataset. The reason being I would then like to initialise a TeViT model with Swin-QueryInst trained on my dataset.

To initialise the Swin backbone in the QueryInst training procedure, I pretrained a segmentation model using Swin + some output segmentation layers then initialise the backbone with the pretrained weights. I set the learning rate of the backbone to be `0.1 * lr` of the query decoders.

However, performance of the QueryInst model is really poor and the segmentation performance as measured through IoU per class degrades significantly from the baseline I trained.

Do you have any suggestions?
